src/routes/categories/containers/CategoriesPage.js

```diff
@@ -147,37 +76,29 @@ export class CategoriesPage extends React.Component<Props> {
   }
 
   render () {
-    try {
-      const category = this.props.categories.getCategoryByUrl(this.props.path)
-      return <div>
-        <Breadcrumbs
-          parents={this.props.categories.getAncestors(category)}
-          locationName={this.getLocationName(this.props.location)} />
-        {this.getContent(category)}
-        <PdfButton href={this.getPdfFetchPath()} />
-      </div>
-    } catch (e) {
+    const categoryModel = this.props.categories.getCategoryByUrl(this.props.categoryPath)
```

Comment:
Maybe we should rename `getCategoryByUrl` to `findCategoryByUrl` since it can return undefined...

Reply:
nope, it throws, it does not return undefined